cfme/tests/candu/test_vm_graph.py

```diff
@@ -17,7 +18,7 @@
     pytest.mark.tier(3),
     test_requirements.c_and_u,
     pytest.mark.usefixtures('setup_provider'),
-    pytest.mark.provider([VMwareProvider, RHEVMProvider, EC2Provider, AzureProvider],
+    pytest.mark.provider([VMwareProvider, RHEVMProvider, EC2Provider, AzureProvider], selector=ONE,
```

Comment:
I think you want ONE_PER_TYPE here, as this only collects and ONE, resulting in collection only against vmware as it matches the required_fields.

Reply:
@mshriver , That's correct.

My intent is to limit the tests to one per type, so I should be using ONE_PER_TYPE here.

I've updated the PR.